app/admin/contributions.rb

```diff
@@ -15,7 +15,12 @@
   member_action :refuse, method: :put, only: :index
 
   batch_action :refuse, if: proc { params[:scope] != "recusado" && params[:scope] != "aprovado" } do |ids|
-    batch_action_collection.find(ids).each { |contribution| contribution.refuse!(current_user.id) if contribution.state == "received" }
+    batch_action_collection.find(ids).each { |contribution|
+      if contribution.state == "received"
```

Comment:
enumerize config in the model has the config `predicate: true` so you will have the predicate method for the contribution called `received?`

https://github.com/brainspec/enumerize#predicate-methods

Reply:
Cool, thanks for letting me know!

Already changed :+1: